### PR TITLE
FakePlugin-STEP2 Implement notification in 2 different ways

### PR DESCRIPTION
**Pull request by nquinquenel**
Add a first extension point to contribute a button in the toolbar. When user clicks on the button, display a notification with “Hello World”.

https://xtranet-sonarsource.atlassian.net/wiki/spaces/SL/pages/2667577345/Fake+SonarLint+Plugin+Specification#Step-2%3A-basic-command-%2B-notification

### CODE REVIEW

File: src/main/kotlin/org/sonar/fakeplugin/notification/NotificationButton.kt [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+
+import com.intellij.notification.NotificationGroupManager
+import com.intellij.notification.NotificationType
+import com.intellij.openapi.project.Project
+
+class NotificationButton {
+
+    companion object {
+        fun notifyText(project: Project, content: String) {
+            NotificationGroupManager.getInstance().getNotificationGroup("Hello World Notification")
```
Comment (henryju) [inline src/main/kotlin/org/sonar/fakeplugin/notification/NotificationButton.kt]: We would normally use a constant for the notification group 'id'

File: src/main/kotlin/org/sonar/fakeplugin/ui/NotificationModal.kt [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+
+class NotificationModal(private val textDescription: String): DialogWrapper(true) {
+
+    init {
+        super.setTitle("Notification Modal")
+        super.init()
+    }
+
+    override fun createCenterPanel(): JComponent {
+        val panel = JPanel(BorderLayout())
```
Comment (henryju) [inline src/main/kotlin/org/sonar/fakeplugin/ui/NotificationModal.kt]: IntelliJ provides customized classes for most Swing components. It is better to use them as much as possible, as they are better integrated: `JBPanel`, `JBLabel`, ...

**Review by henryju — APPROVED** [review]
Minor feedback but overall LGTM!

File: src/main/kotlin/org/sonar/fakeplugin/notification/NotificationButton.kt (hunk 1 above)
Comment (nquinquenel) [reply src/main/kotlin/org/sonar/fakeplugin/notification/NotificationButton.kt]: fixed

File: src/main/kotlin/org/sonar/fakeplugin/ui/NotificationModal.kt (hunk 2 above)
Comment (nquinquenel) [reply src/main/kotlin/org/sonar/fakeplugin/ui/NotificationModal.kt]: done

**Review by henryju — APPROVED** [review]

**Pull request merged**